helm/reana/templates/reana-workflow-controller.yaml

```diff
@@ -91,16 +91,12 @@ spec:
           - name: REANA_RUNTIME_SESSIONS_KUBERNETES_NODE_LABEL
             value: {{ .Values.node_label_runtimesessions }}
           {{- end }}
-          {{- if .Values.kubernetes_jobs_memory_limit }}
-          {{- $_ := required "In order to use .Values.kubernetes_jobs_memory_limit safely you should set the label .Values.node_label_runtime and label a node with it." .Values.node_label_runtime }}
```

Comment:
Can you explain what are the implications of removing `node_label_runtime` check? As I understand from [this comment](https://github.com/reanahub/reana/issues/509#issuecomment-859671843), it would be better to keep it? But probably with this check it won't work? @tiborsimko

Reply:
In principle, memory limits and node labels aren't related at all, so I'm not sure this check belonged here. The main implication is that the deployment wouldn't work by default (contrary to what we have now), as it'd be required to set the node label first.

In addition, the `node_label_runtime` value name was wrong, because now we have `node_label_runtimebatch`, `node_label_runtimejobs` and `node_label_runtimesessions`.